src/styles/transformers/callOnce.js

```diff
@@ -0,0 +1,15 @@
+import warning from 'warning';
+
+const CALLED_ONCE = 'muiPrepared';
+
+export default function callOnce() {
+  if (process.env.NODE_ENV === 'production') return;
```

Comment:
What about englobing the all code inside the `if` statement.
I'm not sure uglify is smart enough to remove unreachable code?